pkg/apiserver/registry/core/seed/strategy_test.go

```diff
@@ -22,6 +22,24 @@ var _ = Describe("Strategy", func() {
 		strategy = Strategy{}
 	)
 
+	Describe("#Canonicalize", func() {
```

Comment:
Cosmetic suggestion: Could we move the Describe block for `#Canonicalize` after the one for `#PrepareForUpdate` so that the Describe blocks follow the order of the funcs in `pkg/apiserver/registry/core/seed/strategy.go`.

Reply:
This comment is not addressed.